about.html

```diff
@@ -0,0 +1,107 @@
+          </div>
+      </div>
+      <div class="mobile-menu hide" id="mobile-menu-id">
+          <button id="cross">
+            <img src="./images/Icon.png" alt="">
+          </button>
+          <nav class="menu-list">
+              <ul>
+                  <li><a href="index.html">Home</a></li>
+                  <li><a href="about.html">About</a></li>
+                  <li><a href="#services-id">program</a></li>
+                  <li><a href="#team-id">sponsor</a></li>
+                  <li><a href="#team-id">news</a></li>
+              </ul>
+          </nav>
+      </div>
+  </nav>
+  <div class="nav-top">
+      <div class="desktop-nav-list" id="nav-top-list">
+          <ul>
+              <li><a href=""><i class="fab fa-facebook-f"></i></a></li>
+              <li><a href=""><i class="fab fa-twitter"></i></a></li>
+              <li><a href="">English</a></li>
+              <li><a href=""> My Page</a></li>
+          </ul>
+      </div>
+  </div>
+  <div class="desktop-navbar">
+      <div class="logo">
+          <img src="./images/riana.png" alt="logo-image">
+      </div>
+      <div class="desktop-nav-list">
+          <ul>
+              <li><a href="about.html">About</a></li>
+              <li><a href="#team-id">program</a></li>
+              <li><a href="#team-id"> Sponsor</a></li>
+              <li><a href="#team-id"> News</a></li>
+          </ul>
+      </div>
+  </div>
+  </nav>
+  <div class="welcome">
+    <h3 class="text">"online live streaming of WORLD EXPO 2030 BUSAN KOREA CONCERT BTS"</h3>
+    <h1 class="text">[LOCATION] Busan Asiad Main Stadium</h1>
+    </h1>
+  </div>
+  <div class="welcome-description">
+    <p class="welcome-text">
+        [LOCATION] Busan Asiad Main Stadium
+    </p>
+    </div>
+    <div class="event-date">
+        <h2 class="fan">don&#39;t miss the show</h2>
+        <p class="fan p">@ BTS, The world rocking <strong>boys</strong></p>
+      </div>
+  </div>
+</div>
+</section>
+<section class="songs">
+    <div class="songs-heading">
+        <p>Our recent completed Albums</p>
+        <p id="about-song">Take a look at our recently completed Albums</p>
+    </div>
+    <div class="album">
+        <div class="song1">
+            <img class="song-img" src="./images/dynimate.png" alt="song1-image">
+            <h4 class="song-detail">This is a situation that everybody else around the world is facing</h4>
+        </div>
+        <div class="song2">
+            <img class="song-img" src="./images/fake love .png" alt="song2-image">
+            <h4 class="song-detail">to struggle with self love and self deprecating thoughts</h4>
+        </div>
+    </div>
+</section>
+<footer class="footer-about">
+    <div class="footer-logo-about">
+        <img src="./images/riana.png" alt="logo-image">
+    </div>
+    <div class="footer-description-about">
+        <p>Seats are available for fans</p>
+    </div>
+</footer>
```

Comment:
- Please Look at the pictures I have provided and Work again on these sections. Kindly refer to the Original design
- Original design:
- 
![Capture](https://user-images.githubusercontent.com/63972670/198004660-4f9ba23a-d08b-4d92-a385-c84019354e4e.PNG)
- Your design:
- 
![Capture](https://user-images.githubusercontent.com/63972670/198005630-680925b4-8c32-40a3-bd2a-62d9e08d6111.PNG)